public/disposable/stylesheet/theme_v3.css

```diff
@@ -131,6 +131,12 @@ a:active {
     color: goldenrod;
 }
 
+#Dispo_NavBar .dropdown-menu a:active {
+  text-decoration: none;
+  background-color: rgba(31, 13, 192, 0.9);
```

Comment:
If we define this as `transparent` then it will match nicely with edited stylesheets, imagine it working like `a:hover`